.github/workflows/sweep.yml

```diff
@@ -0,0 +1,70 @@
+name: ML
+
+on:
+  schedule:
+    # Run this workflow daily at 7:00 UTC
+    - cron: "0 7 * * *"
+  pull_request:
```

Comment:
Enabling manual triggering might be useful in the case where we want to quickly run this workflow.

https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/